### PR TITLE
Clarifies registration of pre-existing domains registered elsewhere

### PR DESCRIPTION
**Pull request by deandre**
2LD Registrations for pre-existing domains registered elsewhere are now supported with Priority Token.

### CODE REVIEW

File: content/articles/domains-au.markdown [hunk 1]
```diff
@@ -28,13 +28,15 @@ To check your priority category, or to see if there are other registrants eligib
 
 If there is no existing priority claim, meaning no matching registrations in any third-level .AU domain, you can register your domain right away, as with [any other domain](/articles/registering-domain).
 
-## Claiming a pre-existing domain
+### Claiming a pre-existing domain
 
-If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) with us, and you want to register the corresponding .AU name, we're planning to add support for this shortly after the start of the priority registration process.
+If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) that's _not_ registered with us, registrations are [now available](/articles/registering-domain). You will need to obtain a [Priority Token](https://priority.auda.org.au/) from auDA.
```
Comment (OleMchls) [inline content/articles/domains-au.markdown:33]: Technically correct, we can also invite them to transfer the domain name to us to use the flow yet to be implemented :)

File: content/articles/domains-au.markdown [hunk 2]
```diff
@@ -28,13 +28,15 @@ To check your priority category, or to see if there are other registrants eligib
...
 
-If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) that's not registered with us, [contact us](https://dnsimple.com/contact), and we'll be happy to help.
+If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) with us, and you want to register the corresponding .AU name, we're planning to add support for this shortly after the start of the priority registration process. [Contact us](https://dnsimple.com/contact), and we'll be happy to help.
```
Comment (OleMchls) [inline content/articles/domains-au.markdown:35]:
```suggestion
If you have an existing third-level .AU domain (.COM.AU, ORG.AU, etc.) with us, and you want to register the corresponding .AU name, we're working on adding support for this shortly. [Contact us](https://dnsimple.com/contact), and we'll be happy to help.
```

**Review by OleMchls — APPROVED** [review]
Thanks for the update! :)

File: content/articles/domains-au.markdown (hunk 1 above)
Comment (deandre) [reply content/articles/domains-au.markdown:33]: I'm hesitant to suggest something for functionality that'a not yet available in a support article.

Comment (OleMchls) [reply content/articles/domains-au.markdown:33]: Understood 👍  So we update as we go 🥳

**Review by itsalyse — APPROVED** [review]

**Pull request merged**